client/src/components/Employees/Employees.js

```diff
@@ -1,13 +1,90 @@
 import React from "react";
 import "./Employees.css";
 import EachEmployee from "./EachEmployee.js";
+import Modal from "react-modal";
+
+function AddEmployee() {
+  function afterOpenModal() {
+    // references are now sync'd and can be accessed.
+    subtitle.style.color = "#black";
+  }
+
+  function openModal() {
+    setIsOpen(true);
+  }
+
+  function closeModal() {
+    setIsOpen(false);
+  }
+
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+
+  var subtitle;
+
+  const customStyles = {
+    content: {
+      top: "50%",
+      left: "50%",
+      right: "auto",
+      bottom: "auto",
+      marginRight: "-50%",
+      height: "45%",
+      width: "32%",
+      transform: "translate(-50%, -50%)"
+    }
+  };
+
+  return (
+    <div>
+      <div className="addbutton">
+        <button onClick={openModal}>+ Add Employees</button>
+      </div>
+      <Modal
+        isOpen={modalIsOpen}
+        onAfterOpen={afterOpenModal}
+        onRequestClose={closeModal}
+        style={customStyles}
+        contentLabel="Example Modal"
+      >
+        <div className="AllText">
+          <h1 className="AddEmpText" ref={_subtitle => (subtitle = _subtitle)}>
+            Add Employee
+          </h1>
+          <div className="EmployeeName">
+            <h3 className="FirstName">First Name</h3>
+            <h3 className="LastName">Last Name</h3>
+            <input className="FirstNameInput" />
+            <input className="LastNameInput" />
+          </div>
+          <div className="EmployeeEmail">
+            <h3 className="Email">Email</h3>
+            <input className="EmailInput" />
+          </div>
+          <h3 className="TrainedFor">Trained For</h3>
+          <div className="RadioButtons">
+            <h3 className="MoffittText">Moffitt</h3>
+            <div className="MoffittButton">
+              <input type="radio" id="moffitt" name="moffitt" value="moffitt" />
+            </div>
+            <h3 className="DoeText">Doe</h3>
+            <div className="DoeButton">
+              <input type="radio" id="doe" name="doe" value="doe" />
+            </div>
+          </div>
+        </div>
+      </Modal>
```

Comment:
We also need a button for them to submit the form and a cancel button for them to get out of the form.

Reply:
done